light/proxy/routes.go

```diff
@@ -92,6 +92,14 @@ func makeGenesisFunc(c *lrpc.Client) rpcGenesisFunc {
 	}
 }
 
+type rpcGenesisChunkedFunc func(ctx *rpctypes.Context, chunkID uint) (*ctypes.ResultGenesisChunk, error)
+
+func makeGenesisChunkedFunc(c *lrpc.Client) rpcGenesisChunkedFunc {
```

Comment:
I think you need to add this to the light client routes further up this page (like you did for `rpc/core/routers.go`